test/conftest.py

```diff
@@ -19,4 +20,15 @@ def default_text_runner(default_rules_collection):
     return RunFromText(default_rules_collection)
 
 
+@pytest.fixture
+def rule_runner(request):
+    """Return runner for a specific rule class."""
+    rule_name = request.param
+    module = import_module(f"ansiblelint.rules.{rule_name}")
+    collection = RulesCollection()
+    rule = getattr(module, rule_name)
```

Comment:
this should be just
```suggestion
    rule = request.param
    collection = RulesCollection()
```

Reply:
This would obviously not work as the rule needs to be loaded and instantiated before being added to the collection.

Reply:
Yes, it's still on the next line. Look closer.

Reply:
Just to reiterate: it's import that I'm against here, the rest of the fixture is great.

Reply:
So basically you want to pass the class object instead of class name to the fixture. Well, that would add one more line to where we use it but I am ok with it, changing it now.

Reply:
Yes, it would add one more line in order to make what's happening explicit and thus more obvious to the test module reader. This fixture, though, may need some sort of input validation but it's not strictly necessary right now.